### ISSUE TITLE
How to run in "release" mode?

### ISSUE THREAD

**Issue by NatElkins** [SAFE-Stack/SAFE-template#319]
The docs give `fake build --target run` as the command to run the application.  I notice this runs `webpack-dev-server` in build.fsx, which appears to be how it's decided whether or not the application is run as a production build or not.

Is there an out of the box command or target to run the application with minification of assets, without HMR, etc?  Or do I need to go in and update `runTool yarnTool "webpack-dev-server" __SOURCE_DIRECTORY__` to just `webpack-server` in build.fsx?

Thanks!

**Comment by NatElkins** [comment]
The motivation for this is that the first thing I did when running this template was checking to see the size of the payload.

<img width="628" alt="image" src="https://user-images.githubusercontent.com/1795462/67872126-e3668380-fb07-11e9-8335-da51b88ab4f2.png">

9.25MB for this simple counter app seems excessive, but I know there's a whole bunch of other stuff getting sent along for the dev experience.

**Comment by kerams** [comment]
I do this in webpack.config.js
```
//var isProduction = !process.argv.find(v => v.indexOf('webpack-dev-server') !== -1);
var isProduction = true;
```

**Comment by isaacabraham** [comment]
Yep, I can see this being useful - especially if you want to check the bundle size locally before deploying to e.g. a test server etc..

What I often do is "steal" the `bundle` build target that you get from SAFE templates built using a `--deploy` flag e.g. `--deploy azure`:

```fsharp
Target.create "Bundle" (fun _ ->
    let serverDir = deployDir
    let publicDir = Path.combine deployDir "public"

    let publishArgs = sprintf "publish -c Release -o \"%s\"" serverDir
    runDotNet publishArgs serverPath

    Shell.copyDir publicDir clientDeployPath FileFilter.allFiles
)
```

This will build and publish the server and generate + emit the client application into a deploy directory from which you can run the server directly. It's not quite the same as what @kerams (rightly) has suggested but sometimes a useful alternative.
